es2015.md

```diff
@@ -1,6 +1,6 @@
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-This document specifies the extensions to the [core ESTree AST types](spec.md) to support the ES6 grammar.
+This document specifies the extensions to the [core ESTree AST types](spec.md) to support the ES2015 grammar.
```

Comment:
That's right. A lot of external links will also now be broken. I'll fix the internal links at least.